src/architecture/clients.md

```diff
@@ -1,10 +1,11 @@
 # Lightweight Clients
 
-To improve system auditability, Exonum includes a [**light
-client**](https://github.com/exonum/exonum-client). Light client is a
-JavaScript library with a number of helper functions available for use by
-frontend developers. These helper functions are used to
-verify blockchain responses on the client side using cryptographic proofs.
+To improve system auditability, Exonum includes a JavaScript library
```

Comment:
Was it necessary to change this paragraph? The new version contains some grammar errors (e.g., "This library..." sentence is ungrammatic). If you don't mind, could you revert it?

Reply:
Have no objections. Done.